data/od/muktaNotification/NotificationTemplateAndActionMapping.json

```diff
@@ -2,6 +2,30 @@
   "tenantId": "od",
   "moduleName": "muktaNotification",
   "NotificationTemplateAndActionMapping": [
+    {
+      "templateCode": "MB_APPROVE",
+      "templateId": "1007665544200114485",
+      "action": "singleSMS",
+      "locale": "en_IN"
+    },
+    {
```

Comment:
For both odiya sms configurations the action will be singlewithbulkunicodeSMS

Reply:
okay Thanks I will update